package.json

```diff
@@ -43,7 +46,6 @@
     "@sentry/vue": "^7.49.0",
     "array-move": "^4.0.0",
```

Comment:
do we still use array-move? out of scope though

Reply:
For swapping share classes/series, I think.

Reply:
Travis, is it OK to resolve this conversation? @seeker25